cypress/e2e/AY11/acessibilidade.cy.js

```diff
@@ -0,0 +1,52 @@
+/// <reference types="cypress" />
+import 'cypress-axe'
+
+describe('ACCESSIBILITY TESTS', () => {
+
+  context('CHECK ACCESSIBLITY IN DARK MODE', () => {
+    beforeEach(() => {
+      cy.visit('/')
+      cy.injectAxe()
+      cy.get('#theme-toggle-button').click()
+      cy.get('[data-theme="dark"]').should('exist')
+    });
+
+    it('CHECK ACCESSIBILITY IN DARK MODE IN DIFFERENT VIEWPORTS', () => {
```

Comment:
Visto que o `context` já descreve que é _dark mode_, repetir isso aqui se torna redundante.

O mesmo vale para os outros testes de A11y.

```suggestion
    it('CHECK ACCESSIBILITY IN DIFFERENT VIEWPORTS', () => {
```